app/views/mailers_common/_booking_confirmation.html.haml

```diff
@@ -114,3 +114,6 @@
 
 %p
   %small Visit ID: #{@visit.visit_id}
+
+%script{type: 'application/ld+json'}
```

Comment:
That could be the case. Alternatively, we can use HTML tags to convey the same information.